src/Contracts/Limiter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Jitendra\PhpValve\Contracts;
+
+interface Limiter
+{
+    /**
+     * Attempts access to a throttled resource.
+     *
+     * Returns following in order:
+     * - Allowed or not?
+     * - X-RateLimit-Limit
+     * - X-RateLimit-Remaining
+     * - X-RateLimit-Reset (in milliseconds)
+     * - X-RateLimit-RetryAfter
+     *
+     * @param  string      $resource
+     * @param  int|integer $worth
+     * @return array
+     */
+    public function attempt(string $resource, int $worth = 1): array;
```

Comment:
why keep default value 1 in the interface?

Reply:
You mean interface need not define defaults?